.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -5,9 +5,10 @@
 **Testing**
 
 1. Have you successfully run `npm run build:release` locally?
-2. How did you test these changes?
-3. Do you add, modify, or delete public API definitions? If yes, has that been reviewed and approved?
-4. Do you change the wire protocol, e.g. the request method? If yes, has that been reviewed and approved?
+2. Can these changes be tested using the demo application? If yes, which demo application can be used to test it?
```

Comment:
Should this be part of the "How did you test these changes" since they are both about testing?
If we do not want to include these then I think it should be after that question.

Reply:
Updated